src/Joomlatools/Console/Command/Site/Install.php

```diff
@@ -121,6 +128,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln("You can login using the following username and password combination: <info>admin</info>/<info>admin</info>.");
     }
 
+    /*
+     * https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment
+     */
+    public function configureDependencies(InputInterface $input, OutputInterface $output)
+    {
+        $path = $this->target_dir;
+
+        $output->writeLn('<info>Installing Joomla 4 project dependencies</info>');
+
+        //first remove existing css, media, node and vendor files
+        exec("rm -rf $path/administrator/templates/atum/css; rm -rf $path/templates/cassiopeia/css; rm -rf $path/media/; rm -rf $path/node_modules/; rm -rf $path/libraries/vendor/; rm -f $path/administrator/cache/autoload_psr4.php;rm -rf $path/installation/template/css");
+
+        exec("composer --working-dir=$path --ignore-platform-reqs install; npm ci --prefix $path");
```

Comment:
@yiendos Also, the npm run is always failing for me (testing on a brand new build of the JT box). Is this really necessary? Why can't we use the existing files?

Reply:
@stevenrombauts because in their infinite wisdom, they've decided to make a beta install as hard as possible for a noobie:
<img width="1308" alt="Screenshot 2020-07-23 at 10 03 18" src="https://user-images.githubusercontent.com/3769817/88269385-d8720580-cccb-11ea-8396-f32d9491bf33.png">

https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment

And requires a version of node/npm that is not available to most